### ISSUE TITLE
Run docker on RaspberryPi

### ISSUE THREAD

**Issue by stal1n274** [ncareau/mikro-watch#21]
Can it be launched on RaspberryPi?

I see this error:

```
standard_init_linux.go:228: exec user process caused: exec format error
standard_init_linux.go:228: exec user process caused: exec format error
standard_init_linux.go:228: exec user process caused: exec format error
standard_init_linux.go:228: exec user process caused: exec format error
standard_init_linux.go:228: exec user process caused: exec format error
```

**Comment by ncareau** [comment]
I have pushed the correct images on dockerhub. Previously I only pushed multiarch iamges on github, but now i'm pushing them on both github and dockerhub. Your config as-is should now work. Can you test it and report the result please ? Thank you.

**Comment by stal1n274** [comment]
Thank you for your efforts. Unfortunately, I have the same issue:
![image](https://user-images.githubusercontent.com/5851581/137647275-8fff426f-ff65-4921-9dca-3a5d06997560.png)

And I cannot connect into this container's console to verify connection to InfluxDB or check other logs.

**Comment by stal1n274** [comment]
However, I deployed this container using the following command:

```
docker run -d --name mikro-watch \
    -e MIKROTIK_IP=192.168.88.1 \
    -e NETWORK_RANGE=192.168.88.1-192.168.88.255 \
    -e INFLUXDB_HOST=172.20.0.1 \
    -e INFLUXDB_USER=<user> \
    -e INFLUXDB_PASS=<pass> \
    -e INFLUXDB_DATABASE=influxdb \
    -e DNS_REVERSE_LOOKUP=false \
    -e DNS_REVERSE_LOOKUP_TRIM_SUFFIX=false \
    ncareau/mikro-watch
```

And there is another error:

```
cURL error 28: Connection time-out (see https://curl.haxx.se/libcurl/c/libcurl-errors.html)

cURL error 28: Connection time-out (see https://curl.haxx.se/libcurl/c/libcurl-errors.html)

cURL error 28: Connection time-out (see https://curl.haxx.se/libcurl/c/libcurl-errors.html)

cURL error 28: Connection time-out (see https://curl.haxx.se/libcurl/c/libcurl-errors.html)
```

I tried to change the address  of InfluxDB host to other IP that is assigned in docker but without any luck.

**Comment by stal1n274** [comment]
I realized today, that this container doesn't have access to any network at all.

**Comment by ncareau** [comment]
Hehe, well that would certainly explain this behavior. Is this something you can/want to change ?

**Comment by stal1n274** [comment]
> Is this something you can/want to change ?

Yes, please. **Could you please test it on RaspberryPi if it is working?** I'm afraid that it is broken. Something is missing in it, that network is not available from it at all.

It would be good to add into this docker container such utility as `net-tools` for debugging. I tried to do this manually but had no success. My config (as you can see I had added only net-tools utility):

```
pi@raspberrypi:~/projects/Mikro-Watch $ cat Dockerfile
FROM php:7.4-cli

RUN apt-get update && apt-get clean && \
            apt-get --allow-insecure-repositories install -y zip net-tools

RUN docker-php-ext-install pcntl bcmath

RUN curl -sS https://getcomposer.org/installer | \
            php -- --install-dir=/usr/bin/ --filename=composer

COPY . /app

WORKDIR /app

RUN php /usr/bin/composer install --no-dev --no-interaction

CMD [ "php", "./mikro-watch",  "daemon" ]
```

Tried to build:

`$ docker build .
Sending build context to Docker daemon  554.5kB
Step 1/8 : FROM php:7.4-cli
 ---> e196894ee135
Step 2/8 : RUN apt-get update && apt-get clean &&           apt-get --allow-insecure-repositories install -y zip net-tools
 ---> Running in 86b72b257a6f
Get:1 http://security.debian.org/debian-security bullseye-security InRelease [44.1 kB]
Get:2 http://deb.debian.org/debian bullseye InRelease [116 kB]
Get:3 http://deb.debian.org/debian bullseye-updates InRelease [39.3 kB]
Err:1 http://security.debian.org/debian-security bullseye-security InRelease
  At least one invalid signature was encountered.
Err:2 http://deb.debian.org/debian bullseye InRelease
  At least one invalid signature was encountered.
Err:3 http://deb.debian.org/debian bullseye-updates InRelease
  At least one invalid signature was encountered.
Reading package lists...
W: GPG error: http://security.debian.org/debian-security bullseye-security InRelease: At least one invalid signature was encountered.
E: The repository 'http://security.debian.org/debian-security bullseye-security InRelease' is not signed.
W: GPG error: http://deb.debian.org/debian bullseye InRelease: At least one invalid signature was encountered.
E: The repository 'http://deb.debian.org/debian bullseye InRelease' is not signed.
W: GPG error: http://deb.debian.org/debian bullseye-updates InRelease: At least one invalid signature was encountered.
E: The repository 'http://deb.debian.org/debian bullseye-updates InRelease' is not signed.
The command '/bin/sh -c apt-get update && apt-get clean &&          apt-get --allow-insecure-repositories install -y zip net-tools' returned a non-zero code: 100
$
`
Seems that there is no chance to build this container directly on my Raspberry
